rollup.config.js

```diff
@@ -8,8 +9,15 @@ export default {
 		{ file: pkg.module, format: 'esm' }
 	],
 	plugins: [
-		sucrase({
-			transforms: ['typescript']
+		resolve({
+			extensions: ['.ts']
+		}),
+		typescript({
+			tsconfigOverride: {
+				compilerOptions: {
+					declaration: false
```

Comment:
Since you explicitly wanted them in the types directory – so have to run `tsc` separately as prebuild